### PR TITLE
[Cherry-pick #9125] Adding dut & dut_asic namespace to getSchedulerParam in qos_sai_base

### PR DESCRIPTION
**Pull request by ansrajpu-git**
### Description of PR

Due to the change in PR(#15914) the search criteria to get scheduler parameters from Redis db needs to be changed
PR# https://github.com/sonic-net/sonic-mgmt/pull/9125 raised for master branch

Summary:
The query to redis db was previously searched with only port & queue . Now in addition to it the duthost & asic namespace has to be concatenated in search
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205

#### What is the motivation for this PR?
The code base is different from master branch, hence raising separate PR for 202205
#### How did you verify/test it?
Re-run the qos test suite

### CODE REVIEW

File: tests/qos/qos_sai_base.py [hunk 1]
```diff
@@ -381,12 +381,31 @@ def __getSchedulerParam(self, dut_asic, port, queue):
...
+                            "redis-cli", "-n", "4", "HGET",
+                            "QUEUE|{0}|{1}|{2}|{3}"
+                            .format(dut_asic.sonichost.hostname, dut_asic.namespace, port, queue), "scheduler"
+                        ]
+                    )[0].encode("utf-8")
+                else:
+                    schedProfile = "SCHEDULER|" + dut_asic.run_redis_cmd(
+                        argv=[
+                            "redis-cli", "-n", "4", "HGET",
+                            "QUEUE|{0}|asic0|{1}|{2}"
```
Comment (kenneth-arista) [inline tests/qos/qos_sai_base.py:398]: For single single systems, I believe the token should be `Asic0`.

Comment (ansrajpu-git) [reply tests/qos/qos_sai_base.py]: Updated the token as suggested. Kindly review

**Comment by vmittal-msft** [comment]
@yxieca please help with merge.

**Comment by vmittal-msft** [comment]
master pr - https://github.com/sonic-net/sonic-mgmt/pull/9125

**Comment by judyjoseph** [comment]
@wangxin please could you merge ?

**Pull request merged**